Variables.js

```diff
@@ -97,9 +101,9 @@ const VARIABLE_TASK_CHASTITY_EXPERIENCE = 'taskChastityExperience';
 const VARIABLE_CHASTITY_TRAININGS_DONE = 'chastityTrainingsDone';
 const VARIABLE_AFRAID_OF_CHASTITY = 'afraidOfChastity';
 const VARIABLE_LAST_CHASTITY_TASK_ID = 'lastChastityTaskId';
+const VARIABLE_CHASTITY_EXPERIENCE_MULTIPLIER = 'chastityEXPMultiplier';
```

Comment:
Why? What?

Reply:
beats me...  I think this must be some of your old code coming back.... I pulled your repository and did a manual merge to get to a working state then pushed back... maybe you removed this in the interim?